Enhancer/EnhancerInterface.php

```diff
@@ -28,8 +27,8 @@
      *
      *     $context->addData('foobar', 'Some value');
      *
-     * @param Context Serialization context
-     * @param resource The resource being serialized
+     * @param []       $data        Context Serialization context
+     * @param Resource $resource    The resource being serialized
      */
-    public function enhance(array $data, PuliResource $resource);
+    public function enhance(array $data, Resource $resource);
```

Comment:
Now I am completely confused. Localy i still got `Resource`. Your Link sais the change was the other way around, and i trust you, so i have the wrong version and i think i did the same wrong change on resource-bundle too. No clue if @dantleech took that error too. But at least we should try to force to get the latest version of that bundle.